docs/credentials.rst

```diff
@@ -0,0 +1,57 @@
+===========
+Credentials
+===========
+
+Oauth2 Client Credentials Plugin
+--------------------------------
+
+.. autoclass:: pyzeebe.credentials.Oauth2ClientCredentialsMetadataPlugin
+    :members:
+    :special-members:
+    :private-members:
+   
+Example:
+
+.. code-block:: python
+
+   oauth2_client_credentials = Oauth2ClientCredentialsMetadataPlugin(
+      client_id=ZEEBE_CLIENT_ID,
+      client_secret=ZEEBE_CLIENT_SECRET,
+      authorization_server=ZEEBE_AUTHORIZATION_SERVER_URL,
+      scope="profile email",
+      audience="zeebe-api",
+   )
+   call_credentials: grpc.CallCredentials = grpc.metadata_call_credentials(oauth2_client_credentials)
+   channel_credentials: grpc.ChannelCredentials = grpc.ssl_channel_credentials(
+      certificate_chain=None, private_key=None, root_certificates=None
+   )
+   composite_credentials: grpc.ChannelCredentials = grpc.composite_channel_credentials(
+      channel_credentials, call_credentials
+   )
+   options: ChannelArgumentType = (("grpc.so_reuseport", 0),)
+   channel: grpc.aio.Channel = grpc.aio.secure_channel(
```

Comment:
`pyzeebe.create_secure_channel`

Reply:
This just documents the most explicit way of using Oauth2ClientCredentialsMetadataPlugin, normally these classes are not used directly anyways, so I think this should be fine.

Reply:
Maybe add a note about default options? (keep-alive)